Cython/Build/Inline.py

```diff
@@ -161,6 +161,13 @@ def cython_inline(code, get_type=unsafe_type,
                   cython_include_dirs=None, cython_compiler_directives=None,
                   force=False, quiet=False, locals=None, globals=None, language_level=None, **kwds):
 
+    from_func_obj = inspect.isfunction(code)
```

Comment:
Isn't it more like "either a string, or something that might have a `__name__` and `__code__` attribute" ?